# Messages
uint8 id
uint32 position
uint8 direction_state